traveler-swift-core/TravelerKit/Models/ParkingItemQuery.swift

```diff
@@ -11,15 +11,15 @@ import Foundation
 /// A query to fetch Parking items
 public struct ParkingItemQuery {
     /// Pagination offset
-    public let offset: Int
+    public var offset: Int
     /// Pagination limit
-    public let limit: Int
+    public var limit: Int
     /// IATA code as string for aiport oriented queries
-    public let airportIATA: String?
+    public var airportIATA: String?
     /// A range of dates where parking is available
-    public let dateRange: ClosedRange<Date>
+    public var dateRange: ClosedRange<Date>
     /// A `BoundingBox` representing the geographic area in which items should be searched for
-    public let boundingBox: BoundingBox?
+    public var boundingBox: BoundingBox?
```

Comment:
Why are these properties all changed to variables?

Reply:
Because the bounding box changes as the person scrolls through the map. If you're already going to let the user modify a variable in a struct, might as well let them modify the rest, unless theres a strict specific reason not to, in this case it makes the code cleaner to update the query by modifiying the variables directly (see its use in the VC that lets you change the parameters).